hack/copyright/BSDandGPL2.txt

```diff
@@ -0,0 +1,2 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
```

Comment:
what's the diff with hack/copyright/BSDandGPL1.txt

Reply:
Because there are two different forms of copyright in Kmesh, two files are used to store.